### PR TITLE
Tornado 4.2.1 is needed for the thin of Py2 minions

### PR DESCRIPTION
**Pull request by brejoc**

### CODE REVIEW

**Comment by brejoc** [comment]
@mcalmer @dincamihai I'm very open for feedback here!

File: salt/py26-compat-salt.spec [hunk 1]
```diff
@@ -456,6 +458,11 @@ cp %{S:5} ./.travis.yml
 %{__python} setup.py --with-salt-version=%{version} --salt-transport=both build
 cp ./build/lib/salt/_version.py ./salt
 
+# Preparations for Tornado version that's needed in the thin file
+tar xfv %{S:7}
+mkdir -p %{buildroot}/opt/tornado/lib/python2.7/site-packages/
```
Comment (mcalmer) [inline salt/py26-compat-salt.spec:463]: Hmm, why not using the compat dir? I think /opt is always wrong for distributors as we are.

**Review by mcalmer — COMMENTED** [review]
I do not like /opt . Please check if compatdir could be used.
This was the idea of that directory. Everything what we need should be there.

**Review by dincamihai — COMMENTED** [review]
I don't have any objections. This can be merged once the open issues are resolved. :+1:

File: salt/py26-compat-salt.spec (hunk 1 above)
Comment (dincamihai) [reply salt/py26-compat-salt.spec:463]: and probably the `tornado` part from the path is not needed

Comment (brejoc) [reply salt/py26-compat-salt.spec:463]: `compatdir` also seems to be the wrong location, since tornado ships with an `.so` file. Maybe `/usr/lib`?

**Comment by brejoc** [comment]
@mcalmer Perhaps you know the answer to this one. Currently we are using `BuildArch: noarch`, which has to be removed since we have that `.so` file now. But when I'm removing that and build it with `osc build`, it's still compiling it as a `noarch` pkg. Shouldn't the resulting pkg have the arch of the build machine?

**Comment by mcalmer** [comment]
yes it should. Please check if there are subpackages and you removed the BuildArch only on one.

**Comment by mcalmer** [comment]
Hmm, btw. I think we need now 2 different packages. I think we should not share this also for 3.2

So SUSE Manager 3.2 and 4.0 needs seperate packages.
Or we stay with the py26-compat-salt shared and build a new package for tornado
`py26-compat-tornado` which can be an arch package

**Comment by brejoc** [comment]
An additional `py26-compat-tornado` pkg would probably be cleaner. Okay, I'm closing this.

**Pull request closed without merging**